Sources/HaCWebsiteLib/Controllers/LandingPageController.swift

```diff
@@ -7,14 +7,15 @@ import DotEnv
 import SwiftyJSON
 
 struct LandingPageController {
-  static let updates = [
-    PostCard(
-      title: "Conquering Linux",
-      category: .workshop,
-      description: "Be great at computers, not because you can, but because you dream.",
-      backgroundColor: "purple",
-      imageURL: "/static/images/workshop.jpg"
-    ),
+
+  static let eventPostCards : [PostCard] = EventServer.getEvents().flatMap{ event in 
+    guard let post = event as? PostCardRepresentable else {
```

Comment:
I think this does the right thing but I would appreciate a swifty person looking over this :wink:

Reply:
Could use chaining here to make the intent clearer:
(event as? PostCardRepresentable)?.postCardRepresentation